### PR TITLE
feat:Added fuseki documentation

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description
Added documentation to build fuseki docker image and updated Helm values.xml.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,17 @@ Run `mvn install` to run unit tests, build and install the package.
 ## Run Package Locally
 To check whether the build was successful, you can start the resulting JAR file from the build process by running `java -jar target/semantic-hub-backend-{current-version}.jar`.
 
+## Precondition
+Build fuseki docker image by following the below steps :
+- Download [jena-fuseki-docker-4.7.0.zip](https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-docker/4.7.0/jena-fuseki-docker-4.7.0.zip)
+- Unzip the jena-fuseki-docker-4.7.0.zip.
+- Build the docker image by running the command - `docker build --build-arg JENA_VERSION=4.7.0 -t jena-fuseki-docker:4.7.0 .`
+
+This docker image `jena-fuseki-docker:4.7.0` will be used in the Helm deployment - [values.yaml](charts/semantic-hub/values.yaml) (graphdb.image).
```
Comment (tunacicek) [inline README.md:38]: Add also the information that this image is used for the tests.

File: README.md [hunk 2]
```diff
@@ -29,11 +29,17 @@ Run `mvn install` to run unit tests, build and install the package.
 ## Run Package Locally
 To check whether the build was successful, you can start the resulting JAR file from the build process by running `java -jar target/semantic-hub-backend-{current-version}.jar`.
 
+## Precondition
```
Comment (tunacicek) [inline README.md:32]: It would be better to add this section before the Build Packages section. 
But it is up to you :)

File: README.md (hunk 1 above)
Comment (shijinrajbosch) [reply README.md]: incorporated the review comments.

File: README.md (hunk 2 above)
Comment (shijinrajbosch) [reply README.md:25]: incorporated the review comments.

**Pull request merged**